frontend/apps/web/app/layout.tsx

```diff
@@ -24,6 +25,33 @@ export default async function RootLayout({
     <html lang="en" suppressHydrationWarning>
       <head />
       <body className="min-h-screen bg-background font-sans antialiased overflow-scroll">
+        <Script
```

Comment:
is this something that should live in all deployments (like folks deploying to OSS on their computers, etc?) or should it only be enabled for neosync cloud?

These should also probably be behind the analytics enabled flag?

The UnifyScriptProvider is a good example of how to conditionally enable these, that way they can also be behind a singular component like `<GoogleAnalyticsProvider />` or something.